app/src/main/java/com/x8bit/bitwarden/data/auth/repository/AuthRepositoryImpl.kt

```diff
@@ -238,6 +241,7 @@ class AuthRepositoryImpl(
         authDiskSource.userStateFlow,
         authDiskSource.userAccountTokensFlow,
         authDiskSource.userOrganizationsListFlow,
+        authDiskSource.userIsUsingKeyConnectorFlow,
```

Comment:
This is more a general question for this block of code, is there any reason to not just add this at the end to keep the pre existing order in the subsequent lines?

Reply:
I was attempting to keep all the `authDiskSource` stuff together.

Reply:
works for me!